cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -610,9 +613,20 @@ class ParquetIOTestBase : public ::testing::Test {
   }
 
   void ReaderFromSink(std::unique_ptr<FileReader>* out) {
+    return ReaderFromSink(out, default_arrow_reader_properties());
+  }
+
+  void ReaderFromSink(std::unique_ptr<FileReader>* out,
+                      const ArrowReaderProperties& arrow_reader_properties) {
```

Comment:
Why not add a default argument value for `arrow_reader_properties` instead of adding a method overload above?